src/fprime_gds/common/pipeline/standard.py

```diff
@@ -93,7 +94,11 @@ def setup(
         # Setup dictionaries encoders and decoders
         self.dictionaries.load_dictionaries(
             self.dictionary_path, packet_spec, packet_set_name
-        )
+        )        
+        # Update config to use Fw types defined in the JSON dictionary
+        if self.dictionaries._fw_type_name_dict:
+            for fw_type_name, fw_type in self.dictionaries._fw_type_name_dict.items():
```

Comment:
We should add and access a `@property` in the `dictionaries` object rather than accessing the `_fw_type_name_dict`.

https://github.com/nasa/fprime-gds/blob/ba05c6d2059506cfd248292026010136aff15c1b/src/fprime_gds/common/pipeline/dictionaries.py#L138-L166